From 13b02a1596a744ed70687dae0ffb465e1979221e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marcus=20R=C3=BCckert?= <darix@nordisch.org>
Date: Tue, 21 Oct 2014 15:13:31 +0200
Subject: [PATCH 09/33] No longer require the ruby version inside the
 subpackage

With the buildrequires we already make sure that the package is only
built if we find a recent enough ABI. then the normal $interpreter(abi)
requires generated by rpm is enough
---
 templates/gem_packages.spec.erb | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/templates/gem_packages.spec.erb b/templates/gem_packages.spec.erb
index 94d4005..29873e5 100644
--- a/templates/gem_packages.spec.erb
+++ b/templates/gem_packages.spec.erb
@@ -77,11 +77,6 @@
   #/ruby2.1
 %>
 %package -n <%= pkg_basename %><%= config[:version_suffix] %>
-<% for req in spec.required_ruby_version -%>
-<%   unless req.empty? -%>
-Requires:       <%= rb_pkgname %> <%= req %>
-<%   end -%>
-<% end -%>
 # MANUAL
 <% if config[:main] && config[:main][:preamble] -%>
 <%= config[:main][:preamble] %>
-- 
2.26.2

